File: Two Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        
        lookup_dict = {}
        
        for position, number in enumerate(nums):
            
            if target - number in lookup_dict:
                return lookup_dict[target-number], position
            
            else:
                lookup_dict[number] = position

File: Two Sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int i;
        int j;
        for (i=1;i<nums.length;i++)
        {
            for (j=i;j<nums.length;j++)
            {
                if (nums[j-i]+nums[j]==target)
                {
                    return new int[]{j-i,j};
                }
            }
        }
        return new int[]{-1,-1};
    }
}
